ovos_utils/gui.py

```diff
@@ -744,7 +744,7 @@ def send_event(self, event_name: str,
     def _pages2uri(self, page_names: List[str]) -> List[str]:
         # Convert pages to full reference
         page_urls = []
-        extra_dirs = [self.resource_dir] or list()
+        extra_dirs = [self.ui_directories.values()] or list()
```

Comment:
should this be a double list? or only self.ui_directories.values() ?

Reply:
Yeah, meant to cast to a list